waves-ext/src/main/container/start-matcher-node.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+NETWORKS="mainnet testnet stagenet"
```

Comment:
Let's add a comment after shebang that we need this file until NODE-2172 is done

Reply:
Done